File: 2000 98 & ME/QuickStart/howto/samples/Services/MessageQueue/formatters/order.cs
using System;

public class Order
{
	protected Int64 orderId;
	protected string itemDescription;
	
	public Int64 OrderId{
		get{
			return this.orderId;
		}
		set{
			this.orderId = value;
		}
	}

	public string ItemDescription{
		get{
			return this.itemDescription;
		}
		set{
			this.itemDescription= value;
		}
	}	
}

File: 2000 98 & ME/QuickStart/howto/samples/Services/MessageQueue/formatters/vb/order.vb
Imports System

Public Class Order
	protected id As Int64
	protected description As String
	
	public property OrderId As Int64
		get
			return me.id
		end get
		set
			me.id = value
		end set
	end property

	public property ItemDescription As String
		get
			return me.description
		end get 
		set
			me.description= value
		end set
        end property
end class